wd_core.au3

```diff
@@ -1589,11 +1589,12 @@ EndFunc   ;==>__WD_Delete
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func __WD_Error($sWhere, $iErr, $sMessage = Default, $iExt = 0)
+Func __WD_Error($sWhere, $iErr, $sMessage = Default, $iExt = Default)
 	Local Const $sFuncName = "__WD_Error"
 	Local $sMsg
 
 	If $sMessage = Default Then $sMessage = ''
+	If $iExt = Default Then $iExt = 0

```

Comment:
currently this is needed, maybe in future we will check other cases

Reply:
I suggest that you leave it set to `default`. Then you could change this line --

`Local $sExtended = (($iExt) ? (" / " & $iExt) : (""))`

to

`Local $sExtended = (($iExt <> Default) ? (" / " & $iExt) : (""))`

Reply:
Take a look on: [$iExt Default refactoring](https://github.com/Danp2/au3WebDriver/pull/244/commits/244d9ee0b0292fed21cf5c22aade1491c1219433)